addon/webextension/selector/ui.js

```diff
@@ -582,7 +588,17 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       this.bgRight.style.width = "100%";
 
       if (this.downloadNotice) {
-        this.downloadNotice.style.bottom = (docHeight - pageYOffset - winBottom + 10) + "px";
+        this.downloadNotice.style.top = (pageYOffset + winBottom - 60) + "px";
+
+        const repositionDownloadNotice = () => {
+          const currentYOffset = window.pageYOffset;
+          const currentWinBottom = window.innerHeight;
+          this.downloadNotice.style.top = (currentYOffset + currentWinBottom - 60) + "px";
+        };
+
+        const windowChangeStop = this.delayExecution(100, repositionDownloadNotice);
+        window.addEventListener('scroll', windowChangeStop);
```

Comment:
Bind events to make it the download notice stick on scroll and resize events. These callbacks are only executed at a maximum every 100ms.